<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gproduits</class>
 <widget class="QDialog" name="Gproduits">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/*-----QWidget-----*/

QWidget

{

	background-color: qlineargradient(spread:repeat, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(27, 39, 50, 255),stop:1 rgba(47, 53, 74, 255));

	color: #000000;



}





/*-----QLabel-----*/

QLabel

{

	background-color: transparent;

	color: #c2c7d5;

	font-size: 13px;



}





/*-----QPushButton-----*/

QPushButton

{

	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));

	color: #fff;

	font-size: 11px;

	font-weight: bold;

	border: none;

	border-radius: 3px;

	padding: 5px;



}





QPushButton::pressed

{

	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 207, 179, 255),stop:1 rgba(70, 255, 230, 255));



}





/*-----QCheckBox-----*/

QCheckBox

{

	background-color: transparent;

	color: #fff;

	font-size: 10px;

	font-weight: bold;

	border: none;

	border-radius: 5px;



}





/*-----QCheckBox-----*/

QCheckBox::indicator

{

    color: #b1b1b1;

    background-color: #323232;

    border: 1px solid darkgray;

    width: 12px;

    height: 12px;



}





QCheckBox::indicator:checked

{

    image:url(&quot;./ressources/check.png&quot;);

	background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));;

    border: 1px solid #607cff;



}





QCheckBox::indicator:unchecked:hover

{

    border: 1px solid #08b099;



}





QCheckBox::disabled

{

	color: #656565;



}





QCheckBox::indicator:disabled

{

	background-color: #656565;

	color: #656565;

    border: 1px solid #656565;



}





/*-----QLineEdit-----*/

QLineEdit

{

	background-color: #c2c7d5;

	color: #000;

	font-weight: bold;

	border: none;

	border-radius: 2px;

	padding: 3px;



}







/*-----QListView-----*/

QListView

{

	background-color: qlineargradient(spread:pad, x1:1, y1:0, x2:1, y2:1, stop:0 rgba(50, 61, 80, 255),stop:1 rgba(44, 49, 69, 255));

	color: #fff;

	font-size: 12px;

	font-weight: bold;

	border: 1px solid #191919;

	show-decoration-selected: 0;



}





QListView::item

{

	color: #31cecb;

	background-color: #454e5e;

	border: none;

	padding: 5px;

	border-radius: 0px;

	padding-left : 10px;

	height: 42px;



}



QListView::item:selected

{

	color: #31cecb;

	background-color: #454e5e;



}





QListView::item:!selected

{

	color:white;

	background-color: transparent;

	border: none;

	padding-left : 10px;



}





QListView::item:!selected:hover

{

	color: #bbbcba;

	background-color: #454e5e;

	border: none;

	padding-left : 10px;



}





/*-----QTreeView-----*/

QTreeView 

{

	background-color: #232939;

	show-decoration-selected: 0;

	color: #c2c8d7;



}





QTreeView::item 

{

	border-top-color: transparent;

	border-bottom-color: transparent;



}





QTreeView::item:hover 

{

	background-color: #606060;

	color: #fff;



}





QTreeView::item:selected 

{

	background-color: #0ab19a;

	color: #fff;



}





QTreeView::item:selected:active

{

	background-color: #0ab19a;

	color: #fff;



}





QTreeView::branch:has-children:!has-siblings:closed,

QTreeView::branch:closed:has-children:has-siblings 

{

	image: url(://tree-closed.png);



}



QTreeView::branch:open:has-children:!has-siblings,

QTreeView::branch:open:has-children:has-siblings  

{

	image: url(://tree-open.png);



}





/*-----QTableView &amp; QTableWidget-----*/

QTableView

{

    background-color: #232939;

	border: 1px solid gray;

    color: #f0f0f0;

    gridline-color: #232939;

    outline : 0;



}





QTableView::disabled

{

    background-color: #242526;

    border: 1px solid #32414B;

    color: #656565;

    gridline-color: #656565;

    outline : 0;



}





QTableView::item:hover 

{

    background-color: #606060;

    color: #f0f0f0;



}





QTableView::item:selected 

{

	background-color: #0ab19a;

    color: #F0F0F0;



}





QTableView::item:selected:disabled

{

    background-color: #1a1b1c;

    border: 2px solid #525251;

    color: #656565;



}





QTableCornerButton::section

{

	background-color: #343a49;

    color: #fff;



}





QHeaderView::section

{

	color: #fff;

	border-top: 0px;

	border-bottom: 1px solid gray;

	border-right: 1px solid gray;

	background-color: #343a49;

    margin-top:1px;

	margin-bottom:1px;

	padding: 5px;



}





QHeaderView::section:disabled

{

    background-color: #525251;

    color: #656565;



}





QHeaderView::section:checked

{

    color: #fff;

    background-color: #0ab19a;



}





QHeaderView::section:checked:disabled

{

    color: #656565;

    background-color: #525251;



}





QHeaderView::section::vertical::first,

QHeaderView::section::vertical::only-one

{

    border-top: 1px solid #353635;



}





QHeaderView::section::vertical

{

    border-top: 1px solid #353635;



}





QHeaderView::section::horizontal::first,

QHeaderView::section::horizontal::only-one

{

    border-left: 1px solid #353635;



}





QHeaderView::section::horizontal

{

    border-left: 1px solid #353635;



}





/*-----QScrollBar-----*/

QScrollBar:horizontal 

{

    background-color: transparent;

    height: 8px;

    margin: 0px;

    padding: 0px;



}





QScrollBar::handle:horizontal 

{

    border: none;

	min-width: 100px;

    background-color: #56576c;



}





QScrollBar::add-line:horizontal, 

QScrollBar::sub-line:horizontal,

QScrollBar::add-page:horizontal, 

QScrollBar::sub-page:horizontal 

{

    width: 0px;

    background-color: transparent;



}





QScrollBar:vertical 

{

    background-color: transparent;

    width: 8px;

    margin: 0;



}





QScrollBar::handle:vertical 

{

    border: none;

	min-height: 100px;

    background-color: #56576c;



}





QScrollBar::add-line:vertical, 

QScrollBar::sub-line:vertical,

QScrollBar::add-page:vertical, 

QScrollBar::sub-page:vertical 

{

    height: 0px;

    background-color: transparent;



}

QGroupBox  {

    border: 1px solid gray;

    border-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));;

    margin-top: 27px;

    font-size: 14px;

    border-radius: 15px;

}



QGroupBox::title  {

    subcontrol-origin: margin;

    subcontrol-position: top center;

    padding: 5px 8000px 5px 8000px;

    background-color: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));;

    color: rgb(255, 255, 255);

}

QLineEdit {

    border: 2px solid gray;

    border-radius: 10px;

    padding: 0 8px;

    background: white;

    selection-background-color: darkgray;

}

QComboBox {

    border: 2px solid grey;

    border-radius: 10px;

    padding: 0 8px;

    min-width: 6em;

}



QComboBox:editable {

    background: white;

}



QComboBox:!editable, QComboBox::drop-down:editable {

     background: white;

}



/* QComboBox gets the &quot;on&quot; state when the popup is open */

QComboBox:!editable:on, QComboBox::drop-down:editable:on {

    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,

                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,

                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);

}



QComboBox:on { /* shift the text when the popup opens */

    padding-top: 3px;

    padding-left: 4px;

}



QComboBox::drop-down {

    subcontrol-origin: padding;

    subcontrol-position: top right;

    width: 15px;



    border-left-width: 1px;

    border-left-color: darkgray;

    border-left-style: solid; /* just a single line */

    border-top-right-radius: 10px; /* same radius as the QComboBox */

    border-bottom-right-radius: 10px;

}



QComboBox::down-arrow {

    image: url(/usr/share/icons/crystalsvg/16x16/actions/1downarrow.png);

}



QComboBox::down-arrow:on { /* shift the arrow when popup is open */

    top: 1px;

    left: 1px;

}

QComboBox QAbstractItemView {

    border: 2px solid gray;

    selection-background-color: lightgray;

}

QTabWidget::pane { /* The tab widget frame */

    border-top: 2px solid #C2C7CB;

}



QTabWidget::tab-bar {

    left: 5px; /* move to the right by 5px */

}



/* Style the tab using the tab sub-control. Note that

    it reads QTabBar _not_ QTabWidget */

QTabBar::tab {

    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,

                                stop: 0 #E1E1E1, stop: 0.4 #DDDDDD,

                                stop: 0.5 #D8D8D8, stop: 1.0 #D3D3D3);

    border: 2px solid qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));;

    border-bottom-color: #C2C7CB; /* same as the pane color */

    border-top-left-radius: 4px;

    border-top-right-radius: 4px;

    min-width: 8ex;

    padding: 2px;

}



QTabBar::tab:selected, QTabBar::tab:hover {

    background: qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));

}



QTabBar::tab:selected {

    border-color:qlineargradient(spread:pad, x1:0, y1:0.511, x2:1, y2:0.511, stop:0 rgba(0, 172, 149, 255),stop:0.995192 rgba(54, 197, 177, 255));

    border-bottom-color: #C2C7CB; /* same as pane color */

}



QTabBar::tab:!selected {

    margin-top: 2px; /* make non-selected tabs look smaller */

}



/* make use of negative margins for overlapping tabs */

QTabBar::tab:selected {

    /* expand/overlap to the left and right by 4px */

    margin-left: -4px;

    margin-right: -4px;

}



QTabBar::tab:first:selected {

    margin-left: 0; /* the first selected tab has nothing to overlap with on the left */

}



QTabBar::tab:last:selected {

    margin-right: 0; /* the last selected tab has nothing to overlap with on the right */

}



QTabBar::tab:only-one {

    margin: 0; /* if there is only one tab, we don't want overlapping margins */

}

QRadioButton{



	color:white;

	font:solid;

}

QDateEdit

{

    background-color: white;

	border: 2px solid grey;

    border-radius: 10px;

    padding: 0 8px;

    spacing: 5px; 

    min-width: 6em;

}





QDateEdit::drop-down {

    image: url(:/new/myapp/cbarrowdn.png);

    width:50px;

    height:15px;

    subcontrol-position: right top;

    subcontrol-origin:margin;

    background-color: white;

   	border: 2px solid grey;

    border-radius: 10px;

    padding: 0 8px;

    spacing: 5px; 

    min-width: 6em;

   spacing: 5px; 

}</string>
  </property>
  <widget class="QTabWidget" name="tab_employee_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>1001</width>
     <height>541</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_employee">
    <attribute name="title">
     <string>Gemployee</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>GProduits</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>441</width>
       <height>171</height>
      </rect>
     </property>
     <property name="title">
      <string>Trier Par:</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>391</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="tout_afficher_netoy">
           <property name="text">
            <string>Tout afficher</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pdf_netoy">
           <property name="text">
            <string>PDF</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QRadioButton" name="tri_nom_netoy">
           <property name="text">
            <string>Nom</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tri_quantite_netoy">
           <property name="text">
            <string>Quantite</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="tri_reference_netoy">
           <property name="text">
            <string>Reference</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>170</y>
       <width>431</width>
       <height>281</height>
      </rect>
     </property>
     <property name="title">
      <string>Les inforamtions du produit</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>381</width>
        <height>211</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Numero:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Nom:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Quantite:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Reference:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="line_num_netoy"/>
           </item>
           <item>
            <widget class="QLineEdit" name="line_nom_netoy"/>
           </item>
           <item>
            <widget class="QLineEdit" name="line_qty_netoy"/>
           </item>
           <item>
            <widget class="QLineEdit" name="line_ref_netoy"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="btn_ajout_netoy">
           <property name="text">
            <string>AJOUTER</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_modif_netoy">
           <property name="text">
            <string>MODIFIER</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_sup_netoy">
           <property name="text">
            <string>SUPPRIMER</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="quitter_netoy">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Quitter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cmd_netoy">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>10</y>
       <width>75</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Commandes</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_info_netoy">
     <property name="geometry">
      <rect>
       <x>46</x>
       <y>464</y>
       <width>901</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>451</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Recherche:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_rech_netoy"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTableView" name="tabNetoy"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>100</y>
       <width>234</width>
       <height>56</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Num:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_modif_netoy"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>type de recherche:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_rech_netoy">
          <item>
           <property name="text">
            <string>Numero ou Quantite</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Nom</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Reference</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="scan_code">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>10</y>
       <width>93</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Scan</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>GArticles</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>GMateriels</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>GClients</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>GFournisseurs</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
